### DISCUSSION TITLE
I call the api.github.com/user interface to get my information, but the name is empty and other parameters are normal. Why?

### DISCUSSION THREAD

**Discussion by Justice996** [community/community#67460] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/27287643/c8cc8924-3e96-449f-bd3c-a714f6c67c78)

**Comment by seyLu** [comment, marked as answer]
#### Hi  @Justice996,

Name in this case, is name field in public profile, not username. 
<table>
<tr>
<td>
<img src="https://github.com/community/community/assets/98249191/a36b2f77-3573-40f0-bead-41498209a8cd">
</td>
<td>
<img src="https://github.com/community/community/assets/98249191/ff530e88-13f7-4c66-9d97-b05ca120d76b">
</td>
</tr>
</table>
<img width="500" src="https://github.com/community/community/assets/98249191/eb06ca10-6609-4ec0-ae69-63031f2d304e">
<img width="500" src="https://github.com/community/community/assets/98249191/44c0b347-96ce-4933-88a7-f4e61173f328">
<img width="500" src="https://github.com/community/community/assets/98249191/5c8b8076-3d6c-48bc-8136-4ab881adc32d">

**Reply by Justice996** [reply]
ok thanks
